packages/core/src/Stores/client-store.js

```diff
@@ -1720,10 +1723,11 @@ export default class ClientStore extends BaseStore {
             }
 
             if (this.account_settings) this.setPreferredLanguage(this.account_settings.preferred_language);
-            this.loginid !== 'null' && RudderStack.setAccountType(this.loginid.substring(0, 2));
+            this.loginid !== 'null' && Analytics.setAttributes({ account_type: this.loginid.substring(0, 2) });
             if (this.user_id) {
-                RudderStack.identifyEvent(this.user_id, {
-                    language: getLanguage().toLowerCase(),
+                Analytics.setAttributes({
+                    user_language: getLanguage().toLowerCase(),
+                    user_id: this.user_id,
```

Comment:
already set in 1648-1651 lines

Reply:
I think this is fine and also in the other places as well, the values are getting changed so it should be updated for our rudderstack instance

Reply:
is user_id always the same? or by account_type change id become different?

Reply:
This is running on `init` function, hence this is also needed along with the `app-contents.tsx` one